### PR TITLE
webui tag operations without skeleton

### PR DESCRIPTION
**Pull request by haribhandari07**
## Description

Do not use skeleton files for ```webUITags```  to speed up CI
## Related Issue

Part of https://github.com/owncloud/QA/issues/622

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] Backport (if applicable set "backport-request" label and remove when the backport was done)

### CODE REVIEW

**Review by dpakach — COMMENTED** [review]
LGTM :+1:

**Review by phil-davis — APPROVED** [review]

**Review by individual-it — CHANGES_REQUESTED** [review]
mostly good, just one small issue

File: tests/acceptance/features/webUITags/deleteTags.feature [hunk 1]
```diff
@@ -5,75 +5,82 @@ Feature: Deletion of existing tags from files and folders
...
-    And the user has added tag "lorem" to file "lorem-big.txt"
-    And the user has browsed directly to display the details of file "lorem.txt" in folder "simple-folder"
-    When the user deletes tag with name "lorem" using the webUI
-    Then tag "lorem" should not exist for user "user1"
+    Given user "user1" has uploaded file with content "some content" to "/randomfile.txt"
+    And user "user1" has uploaded file with content "more content" to "/randomfile-big.txt"
+    And the user has created a "normal" tag with name "random"
+    And the user has added tag "random" to file "randomfile.txt"
+    And the user has added tag "random" to file "randomfile-big.txt"
+    And the user has browsed directly to display the details of file "randomfile.txt" in folder ""
```
[the remark this answers is not in the document]
Comment (phil-davis) [reply tests/acceptance/features/webUITags/deleteTags.feature]: PR is already merged. @HariBhandari07 please sort this out with @individual-it and make another PR to adjust.

**Comment by HariBhandari07** [comment]
Backport on #35497

**Pull request merged**